_specification-v1/sass.md

```diff
@@ -96,11 +96,11 @@ As Sass has limited encapsulation, any Sass (e.g. mixin, function, variable) tha
 
 ## Sass Includes
 
-If a component contains SCSS files other than the main file listed in `bower.json`:
+If a component contains SCSS files other than the root `main.scss` file:
```

Comment:
I can't find a *must* for the existence of `main.scss` or what it should contain.

If this doesn't exist in the spec, I think we should add it in now